### PR TITLE
chore: remove leading space when creating schematics

### LINKED ISSUE #306: Schematics adding extra space to dependencies
Possibly just need to remove the space after the colon from the schematic? `'@terminus/foo: ^3.1.0'`

### PR DESCRIPTION
**Pull request by atlwendy**
closes #306

### CODE REVIEW

File: libs/ui/login-form/schematics/ng-add/index.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export const ngAdd = () => (tree: Tree, context: SchematicContext): Tree => {
     const nodeDependency: NodeDependency = {
       type: NodeDependencyType.Default,
       name: individualPackage[0],
-      version: individualPackage[1],
+      version: individualPackage[1].trim().trim(),
```
Comment (benjamincharity) [inline libs/ui/login-form/schematics/ng-add/index.ts]: I think this may be an extra paste..

**Review by benjamincharity — CHANGES_REQUESTED** [review]
One extra trim, otherwise looks great

File: libs/ui/login-form/schematics/ng-add/index.ts (hunk 1 above)
Comment (atlwendy) [reply libs/ui/login-form/schematics/ng-add/index.ts]: 🤦 I tried one and then used `sed` to do the rest, but forgot to revert the first testing case..
good catch!

**Review by benjamincharity — APPROVED** [review]
Reminder to re-request a review once changes are made - I didn't know this was ready

**Pull request merged**